bot/exts/utils/snekbox.py

```diff
@@ -244,9 +313,11 @@ async def format_output(self, output: str) -> Tuple[str, Optional[str]]:
 
         return output, paste_link
 
+    @lock_arg("snekbox.run_job", "ctx", attrgetter("author.id"), raise_error=True)
```

Comment:
Not really important but...
```suggestion
    @lock_arg("snekbox.send_job", "ctx", attrgetter("author.id"), raise_error=True)
```

Reply:
Good point, fixed this!